lib/node_modules/@stdlib/blas/ext/base/zfill/src/main.c

```diff
@@ -19,27 +19,41 @@
 #include "stdlib/blas/ext/base/zfill.h"
 #include "stdlib/complex/float64/ctor.h"
 #include "stdlib/strided/base/stride2offset.h"
-#include <stdint.h>
+#include "stdlib/blas/base/shared.h"
 
 /**
 * Fills a double-precision complex floating-point strided array with a specified scalar constant.
 *
-* @param N       number of indexed elements
-* @param alpha   scalar constant
-* @param X       input array
-* @param stride  index increment
+* @param N        number of indexed elements
+* @param alpha    scalar constant
+* @param X        input array
+* @param strideX  strided length
 */
-void c_zfill( const int64_t N, const stdlib_complex128_t alpha, stdlib_complex128_t *X, const int64_t stride ) {
-	int64_t ix;
-	int64_t m;
-	int64_t i;
+void API_SUFFIX(stdlib_strided_zfill)( const CBLAS_INT N, const stdlib_complex128_t alpha, stdlib_complex128_t *X, const CBLAS_INT strideX ) {
+	CBLAS_INT ox = stdlib_strided_stride2offset( N, strideX );
+	API_SUFFIX(stdlib_strided_zfill_ndarray)( N, alpha, X, strideX, ox );
+}
+
+/**
+* Fills a double-precision complex floating-point strided array with a specified scalar constant using alternative indexing semantics.
+*
+* @param N        number of indexed elements
+* @param alpha    scalar constant
+* @param X        input array
+* @param strideX  index increment
+* @param offsetX  starting index
+*/
+void API_SUFFIX(stdlib_strided_zfill_ndarray)( const CBLAS_INT N, const stdlib_complex128_t alpha, stdlib_complex128_t *X, const CBLAS_INT strideX, const CBLAS_INT offsetX ) {
+ CBLAS_INT ix;
+ CBLAS_INT m;
+ CBLAS_INT i;
```

Comment:
The indentation is off here.
```suggestion
	CBLAS_INT ix;
	CBLAS_INT m;
	CBLAS_INT i;
```